systems/billing/api-gateway/pkg/client/invoice_test.go

```diff
@@ -125,23 +124,23 @@ func TestBillingClient_AddInvoice(t *testing.T) {
 		}
 
 		invoiceResp := &pb.AddResponse{Invoice: &pb.Invoice{
-			Id:           uuid.NewV4().String(),
-			SubscriberId: "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
-			NetworkId:    uuid.NewV4().String(),
+			Id:         uuid.NewV4().String(),
+			InvoiceeId: "5eb02857-a71e-4ea2-bcf9-57d3a41bc6ba",
```

Comment:
This should be invoiceId I guess .

Reply:
No this is correct. It is invoiceeId. Id is already the invoice id.